task/backend/coordinator.go

```diff
@@ -0,0 +1,43 @@
+package backend
+
+import (
+	"context"
+
+	"github.com/influxdata/influxdb"
+)
+
+// Tasksfinder is a type on which tasks can be listed
+type TasksFinder interface {
+	FindTasks(context.Context, influxdb.TaskFilter) ([]*influxdb.Task, int, error)
+}
+
+// Coordinator is a type with a single method which
+// is called when a task has been created
+type Coordinator interface {
+	TaskCreated(context.Context, *influxdb.Task) error
+}
+
+// NotifyCoordinatorOfCreated lists all tasks by the provided task service and for
+// each task it calls the provided coordinators task created method
+func NotifyCoordinatorOfCreated(ctx context.Context, ts TasksFinder, coord Coordinator) error {
+	// If we missed a Create Action
+	tasks, _, err := ts.FindTasks(ctx, influxdb.TaskFilter{})
```

Comment:
A new addition here that was missed before. We will probably need to set the wildcard on the filter so we pick up "checks and notifications" `influxdb.TaskFilter{Type: & influxdb.TaskTypeWildcard}`.